documentation/allclasses-frame.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
external javadoc should not be generated in this repo
It makes tracking actual changes more difficult.